src/mds/InoTable.cc

```diff
@@ -111,6 +113,8 @@ void InoTable::replay_alloc_id(inodeno_t id)
 }
 void InoTable::replay_alloc_ids(interval_set<inodeno_t>& ids) 
 {
+  assert(mds);  // Only usable in online mode
+
```

Comment:
Ah yes, it was just that I checked which functions used the MDS pointer, and added assertions to those.